docs/modules/ROOT/pages/backup-restore.adoc

```diff
@@ -107,77 +128,100 @@ kubectl create secret generic <secret-name> \
 ----
 --
 ====
-<1> The bucket URI where backup data will be stored in
-<2> Name of the secret with credentials for accessing the given Bucket URI.
+<1> The Bucket URI in which to store backup data.
+<2> Name of the secret with credentials for accessing the Bucket URI.
 
-== Scheduling Backups
+[scheduling-backups]
+== Schedule Backups
 
-You can schedule backups using the `schedule` and `hotBackupTemplate` fields of the `CronHotBackup` resource. For more information about the `CronHotBackup` resource, see the xref:api-ref.adoc#cronhotbackupspec[API Reference].
+You can schedule backups using the `schedule` and `hotBackupTemplate` fields of the `CronHotBackup` resource as follows:
 
 [source,yaml,subs="attributes+"]
 ----
 include::ROOT:example$/cron-hot-backup.yaml[]
 ----
 
+For further information on the `CronHotBackup` resource, see xref:api-ref.adoc#cronhotbackupspec[CronHotBackupSpec] in the Hazelcast Operator API Reference.
+
 The `schedule` field takes a valid cron expression. For example, you can configure the following scheduled backups:
 
 [cols="1,1"]
 |===
+|`schedule` Value|Description
+
 | ```30 10 * * *```
-| At 10:30 AM every day
+| Take a backup at 10:30 AM every day
 
 | ```0, 0, 1,15,25 * *```
-| On 1st, 15th, and 25th of each month at midnight
+| Take a backup at midnight on the 1st, 15th, and 25th of each month
 
 | ```@monthly```
-| On the first day of each month at midnight
+| Take a backup at midnight on the first day of each month
 |===
 
+For a full list of supported expressions, refer to link:https://pkg.go.dev/github.com/robfig/cron/v3@v3.0.1#hdr-CRON_Expression_Format[CRON Expression Format,window=_blank] in the library documentation.
 
-For a full list of supported expressions, see the link:https://pkg.go.dev/github.com/robfig/cron/v3@v3.0.1#hdr-CRON_Expression_Format[library documentation].
+[checking-the-status-of-a-backup]
+== Check the Status of a Backup
 
-== Checking the Status of a Backup
+To check the status of a local backup, use the following command:
 
-To check the status of a local backup, run the following command:
 [source,bash]
 ----
 kubectl get hotbackup hot-backup
 ----
 
-The status of the backup is displayed in the output.
+The result is similar to the following:
+
 [source,bash]
 ----
 NAME         STATUS
 hot-backup   Success
 ----
 
-== Restoring from Local Backups
+[restoring-from-local-backup]
+== Restore from Local Backups
 
-To restore a cluster from local backups, you can directly reapply the `Hazelcast` resource, which gives the cluster access to the PVCs that contain the persisted data. This will restore the Hazelcast cluster from existing `hot-restart` folders.
+You can only use a local backup once to restore a cluster.
```

Comment:
Because a Warning is used to tell users not to do something; or that something is irreversible. You could argue that using the backup means that it cannot be used again; and this is irreversible. However, a recommendation definitely doesn't belong in a warning. 

Also, it is preferable that big orange warnings are minimised on a page for user confidence.

Happy to re-introduce the recommendation, but I'd suggest (also) starting the Restore from External Backups with a similar sentence (which I obviously forgot to do in the initial updated), such as: 

You can use an external backup to persistently restore a backup across clusters.

Reply:
Added suggested sentence to Restore from External Backups